numpydoc/tests/test_docscrape.py

```diff
@@ -216,6 +235,38 @@ def test_yields():
         assert desc[0].endswith(end)
 
 
+def test_sent():
+    section = doc_sent['Receives']
+    assert_equal(len(section), 2)
```

Comment:
CI isn't happy: `NameError: global name 'assert_equal' is not defined`

You can just change this to plain `assert`

Reply:
and some more of the same several lines below